extensions/amp-inline-gallery/1.0/storybook/Basic.js

```diff
@@ -30,7 +30,7 @@ export default {
 export const _default = () => {
   const width = number('width', 440);
   const height = number('height', 225);
-  const paginationHeight = number('top indicator height', 20);
+  const paginationHeight = number('top indicator height', 40);
```

Comment:
Is there a reason for this change?

Reply:
Just easier to test since the default is also `20` so this was showing an immediate difference. I can revert if you prefer.

Reply:
Actually, I went ahead and reverted. I think default value there is clearer.